// [[Reason: TODO: Sourcemap seems missaligned]] ////

//// [tests/cases/compiler/declarationEmitReusesLambdaParameterNodes.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [index.d.ts.map]
-{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,KAAK,EAAE,MAAM,cAAc,CAAC;AAErC,eAAO,MAAM,aAAa,cAAgB,MAAM,MAAM,CAAC,GAAG,EAAE,KAAG,IAAU,CAAA;AACzE,wBAAgB,aAAa,CAAC,MAAM,EAAG,CAAC,EAAE,KAAK,CAAC,MAAM,CAAC,GAAG,EAAE,GAAG,IAAI,CAAG"}
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,KAAK,EAAE,MAAM,cAAc,CAAC;AAErC,eAAO,MAAM,aAAa,GAAI,MAAM,EAAG,CAAC,EAAE,KAAK,CAAC,MAAM,CAAC,GAAG,EAAE,KAAG,IAAU,CAAA;AACzE,wBAAgB,aAAa,CAAC,MAAM,EAAG,CAAC,EAAE,KAAK,CAAC,MAAM,CAAC,GAAG,EAAE,GAAG,IAAI,CAAG"}
 
-//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgUHJvcHMgfSBmcm9tICJyZWFjdC1zZWxlY3QiOw0KZXhwb3J0IGRlY2xhcmUgY29uc3QgQ3VzdG9tU2VsZWN0MTogPE9wdGlvbj4oeDogUHJvcHM8T3B0aW9uPiAmIHt9KSA9PiB2b2lkOw0KZXhwb3J0IGRlY2xhcmUgZnVuY3Rpb24gQ3VzdG9tU2VsZWN0MjxPcHRpb24+KHg6IFByb3BzPE9wdGlvbj4gJiB7fSk6IHZvaWQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxLQUFLLEVBQUUsTUFBTSxjQUFjLENBQUM7QUFFckMsZUFBTyxNQUFNLGFBQWEsY0FBZ0IsTUFBTSxNQUFNLENBQUMsR0FBRyxFQUFFLEtBQUcsSUFBVSxDQUFBO0FBQ3pFLHdCQUFnQixhQUFhLENBQUMsTUFBTSxFQUFHLENBQUMsRUFBRSxLQUFLLENBQUMsTUFBTSxDQUFDLEdBQUcsRUFBRSxHQUFHLElBQUksQ0FBRyJ9,aW1wb3J0IHsgUHJvcHMgfSBmcm9tICJyZWFjdC1zZWxlY3QiOwoKZXhwb3J0IGNvbnN0IEN1c3RvbVNlbGVjdDEgPSA8T3B0aW9uLD4oeDogUHJvcHM8T3B0aW9uPiAmIHt9KTogdm9pZCA9PiB7fQpleHBvcnQgZnVuY3Rpb24gQ3VzdG9tU2VsZWN0MjxPcHRpb24sPih4OiBQcm9wczxPcHRpb24+ICYge30pOiB2b2lkIHt9Cg==
+//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgUHJvcHMgfSBmcm9tICJyZWFjdC1zZWxlY3QiOw0KZXhwb3J0IGRlY2xhcmUgY29uc3QgQ3VzdG9tU2VsZWN0MTogPE9wdGlvbj4oeDogUHJvcHM8T3B0aW9uPiAmIHt9KSA9PiB2b2lkOw0KZXhwb3J0IGRlY2xhcmUgZnVuY3Rpb24gQ3VzdG9tU2VsZWN0MjxPcHRpb24+KHg6IFByb3BzPE9wdGlvbj4gJiB7fSk6IHZvaWQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxLQUFLLEVBQUUsTUFBTSxjQUFjLENBQUM7QUFFckMsZUFBTyxNQUFNLGFBQWEsR0FBSSxNQUFNLEVBQUcsQ0FBQyxFQUFFLEtBQUssQ0FBQyxNQUFNLENBQUMsR0FBRyxFQUFFLEtBQUcsSUFBVSxDQUFBO0FBQ3pFLHdCQUFnQixhQUFhLENBQUMsTUFBTSxFQUFHLENBQUMsRUFBRSxLQUFLLENBQUMsTUFBTSxDQUFDLEdBQUcsRUFBRSxHQUFHLElBQUksQ0FBRyJ9,aW1wb3J0IHsgUHJvcHMgfSBmcm9tICJyZWFjdC1zZWxlY3QiOwoKZXhwb3J0IGNvbnN0IEN1c3RvbVNlbGVjdDEgPSA8T3B0aW9uLD4oeDogUHJvcHM8T3B0aW9uPiAmIHt9KTogdm9pZCA9PiB7fQpleHBvcnQgZnVuY3Rpb24gQ3VzdG9tU2VsZWN0MjxPcHRpb24sPih4OiBQcm9wczxPcHRpb24+ICYge30pOiB2b2lkIHt9Cg==
 
